styles/widgets/forms.less

```diff
@@ -104,7 +107,8 @@ html.ms .propform {
             }
         }
 
-        .form-check-input {
+        .form-check-input,
+        .form-radio-input {
```

Comment:
The same here. https://getbootstrap.com/docs/4.0/components/forms/#checkboxes-and-radios